src/port.js

```diff
@@ -0,0 +1,17 @@
+class Port {
+  constructor(name) {
+      this.name = name
+      this.ships = []
+  }
+  
+  addShip(ship){
+      this.ships.push(ship)
+  }
+
+  removeShip(ship) {
+     const ind = this.ships.indexOf(ship)
+     this.ships.splice(ind, 1)
+  }
+}
+
+module.exports = Port;
```

Comment:
Code looks good, but there is some inconsistent indentation. Sometimes you are indenting once and then other times twice.